MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,169 @@
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        CommitMode="PropertyChanged">
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.CommitMode = DataFormCommitMode.PropertyChanged;
+{% endhighlight %}
+{% endtabs %}
+
+#### Manual
+
+If the commit mode is `Manual`, the value should be committed manually by calling the [SfDataForm.Commit] method.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+        xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        CommitMode="Manual">
+    </dataForm:SfDataForm>
+
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.CommitMode = DataFormCommitMode.Manual;
+{% endhighlight %}
+{% endtabs %}
+
+The following code commits the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight C# %}
+this.dataForm.Commit();
+{% endhighlight %}
+{% endtabs %}
+
+N> On manual commit, manual validation will be called to validate the properties before the commit.
+
+## Value converter attribute
+
+To show the original value in different format or as different value, use the [DataFormValueConverter] attribute.
+
+#### Changing original value of the DataForm property value using converter
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataFormValueConverter(typeof(StringToDateTimeConverter))]
+    public string DateTime { get; set; }
+
+public class StringToDateTimeConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        if (value == null || string.IsNullOrEmpty(value.ToString()))
+        {
+            return DateTime.Now;
+        }
+        else
+        {
+            DateTime dateTime;
+            DateTime.TryParse((string)value, out dateTime);
+            return dateTime;
+        }
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return value.ToString();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the editor will display the original value as `DateTime`. While committing, it is converted to `string` and stored in data object.
+
+## Read only mode
+
+`DataForm` can be disabled by setting the [IsReadOnly] property of the data form.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm" 
+        IsReadOnly="True">
+    </dataForm:SfDataForm>
+
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.IsReadOnly = true;
+{% endhighlight %}
+{% endtabs %}
+
+The particular editor can disabled by setting the [IsReadOnly] property of the [DataFormItem].
```

Comment:
The particular editor can be disabled using attributes and [GenerateDataFormItem] event.

#### Using attribute
An editor can be disabled using [EditableAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.editableattribute?view=net-7.0) and [ReadOnlyAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.readonlyattribute?view=net-6.0)
```

{% tabs %}
{% highlight C# %}

[Editable(false)]
 public string Name { get; set; }

[ReadOnly(true)]
 public string Password { get; set; }

{% endhighlight %}
{% endtabs %}
```


#### Using event
An editor can be disabled by setting the [IsReadOnly] property of the [DataFormItem].

```
{% tabs %}
{% highlight C# %}
this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;

 private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
{
    if (e.DataFormItem != null)
    {
        if (e.DataFormItem.FieldName == "Salary")
            e.DataFormItem.IsReadOnly = true;
    }
}
{% endhighlight %}
{% endtabs %}
```

Reply:
Changes done